src/tests/FlapJob-integration.t.sol

```diff
@@ -0,0 +1,95 @@
+// SPDX-License-Identifier: AGPL-3.0-or-later
+// Copyright (C) 2021 Dai Foundation
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+pragma solidity 0.8.13;
+
+import "./DssCronBase.t.sol";
+
+import {FlapJob} from "../FlapJob.sol";
+
+contract FlapJobIntegrationTest is DssCronBaseTest {
+    using stdStorage for StdStorage;
+    using GodMode for *;
+
+    FlapJob flapJob;
+
+    event LogNote(
+        bytes4   indexed  sig,
+        address  indexed  usr,
+        bytes32  indexed  arg1,
+        bytes32  indexed  arg2,
+        bytes             data
+    ) anonymous;
+
+    function setUpSub() virtual override internal {
+        flapJob = new FlapJob(address(sequencer), address(dss.vat), address(dss.vow));
+
+        // Make sure that if a flapper has a cooldown period it already passed
+        GodMode.vm().warp(block.timestamp + 10 days);
```

Comment:
Yes, currently the test works with the existing flapper, which doesn't even have that.
Was thinking whether to use a mock flapper but opted to avoid mixing mock stuff in an integration test if possible.